tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -91,6 +97,8 @@ Define {
 	BT_PB_HOST_PIPELINE_SINK "copier.SSP.10.1"
 	BT_PB_DAI_PIPELINE_SRC "copier.host.9.1"
 	BT_PB_PIPELINE_STREAM_NAME "copier.SSP.10.1"
+	AEC				0
```

Comment:
What is this? A lot of the other defines are a lot more verbose.

Reply:
sure, updated to GOOGL_RTC_AEC_SUPPORT